cross/fish/PLIST

```diff
@@ -0,0 +1,15 @@
+bin:bin/fish
+bin:bin/fish_indent
+bin:bin/fish_key_reader
+rsc:etc/fish/
+rsc:share/doc/fish/
+rsc:share/fish/__fish_build_paths.fish
+rsc:share/fish/completions/
+rsc:share/fish/config.fish
+rsc:share/fish/functions/
+rsc:share/fish/tools/
+rsc:share/fish/vendor_completions.d/
+rsc:share/fish/vendor_conf.d/
+rsc:share/fish/vendor_functions.d/
+rsc:share/locale/
+rsc:share/pkgconfig/
```

Comment:
sorry, didn't catch this in the first review:

pkgconfig folder should not be installed as it is used at build time only (for other packages that link to this)